### PR TITLE
Add Redux one-day

### PR DESCRIPTION
**Pull request by ric9176**
In order for the PR to not get too big as there are many changes happening I wanted to start with just adding one of the new workshops. This PR adds the new function `getNextTrainingByTrainingID` and makes the relevant updates to handle the new type "React Workshop" in addition to adding the landing and instance for Redux.

### CODE REVIEW

File: src/pages/react/training/workshops/redux/index.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  selectUpcomingTrainings,
+  getNextTrainingByTrainingID,
+} from 'src/components/training'
+import { Image } from 'src/components/elements'
+import { Link, Breadcrumb } from 'src/components/navigation'
+import LinkButton from 'src/components/buttons/LinkButton'
+import { ONE_DAY_WORKSHOP, LONDON } from 'src/config/data'
+import { LIST_TWO_COL } from 'src/components/curriculum/selectCurriculumLayout'
+
+const StylingDesignSystemWorkshop = () => (
```
Comment (alexlbr) [inline src/pages/react/training/workshops/redux/index.js]: @ric9176  StylingDesignSystemWorkshop ?

File: src/components/training/UpcomingTrainings.js [hunk 2]
```diff
@@ -45,6 +46,8 @@ const createTrainingPath = ({ type, city = '', index, id }) => {
       return `/community/meetups/${id}`
     case ONE_DAY_WORKSHOP:
       return `/react/training/workshops/design-system-styling-in-react/`
+    case REACT_WORKSHOP:
+      return `/react/training/workshops/${city.toLowerCase()}/${i}`
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/components/training/UpcomingTrainings.js]: good idea! I'll implement this :-D

File: src/components/training/index.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ export {
   selectUpcomingTrainings,
   selectNthTraining,
   selectTrainingById,
+  getNextTrainingByTrainingID,
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/components/training/index.js]: yes I'll update this, I do wonder wether we should change the other one to `selectTrainingByInstanceId`  as now we are referencing 2 different ID's from the api (training instance and training)

File: src/pages/react/training/workshops/redux/index.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+import { CurriculumOneDayRedux } from 'src/components/curriculum/workshops'
+import { Card } from 'src/components/elements'
+import { HideComponentsUsingCss } from 'src/components/utils'
+import Header from 'src/components/layout/Header'
+import { BOOTCAMP_COLLAB, CATALIN } from 'src/config/images'
+import { CallToActionRow } from 'src/components/layout/CallToActionNextTrainings'
+import {
+  UpcomingTrainingSection,
+  AttendeeQuote,
+  selectUpcomingTrainings,
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/pages/react/training/workshops/redux/index.js]: my bad, it does I just didn't see it!

File: src/pages/react/training/workshops/redux/index.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+            training={training}
+          />
+          <TopSection top>
+            <Grid>
+              <CallToActionRow left>
+                <Col mdOffset={1} md={4}>
+                  <LinkButton
+                    variant="primary"
+                    to="/react/training/workshops/redux/london"
+                    children="Next workshop: 10 July, London"
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/pages/react/training/workshops/redux/index.js]: yes, thanks for spotting!

File: src/components/training/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (alexlbr) [reply src/components/training/index.js]: @ric9176 I think it's a very good idea to call it selectTrainingByInstanceId, it's confusing otherwise

**Pull request merged**